jbpm-wb-executor-service/jbpm-wb-executor-service-api/src/main/java/org/jbpm/workbench/es/model/ExecutionErrorSummary.java

```diff
@@ -0,0 +1,283 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.es.model;
+
+import java.util.Date;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jboss.errai.databinding.client.api.Bindable;
+import org.jbpm.workbench.common.model.GenericSummary;
+import org.jbpm.workbench.es.util.ExecutionErrorType;
+
+@Bindable
+@Portable
+public class ExecutionErrorSummary extends GenericSummary {
+
+    protected String errorId;
+    protected ExecutionErrorType type;
+    protected String deploymentId;
+    protected Long processInstanceId;
+    protected String processId;
+    protected Long activityId;
+    protected String activityName;
+    protected Long jobId;
+    protected String errorMessage;
+    protected String error;
```

Comment:
Perhaps `errorInfo` (with the corresponding `set/getErrorInfo()` methods) would be more descriptive